app/concepts/reservations/use_cases/find_reservations.rb

```diff
@@ -0,0 +1,23 @@
+module Reservations
+  module UseCases
+    class FindReservations
+      def initialize(params:)
+        @params = params
+      end
```

Comment:
I can see you are using only `seance_id` here so it would be better to pass only this param to the use_case.
```suggestion
      def initialize(seance_id:)
        @seanse_id = seance_id
      end
```

Reply:
My idea was to make it more flexible. In the future I could need different query with different param so it would be easier to implement it.